pom.xml

```diff
@@ -61,6 +61,7 @@
         <hibernate.validator.version>6.0.16.Final</hibernate.validator.version>
         <immutables.version>2.7.1</immutables.version>
         <jackson.version>2.9.8</jackson.version>
+        <jackson.databind.version>2.9.9.1</jackson.databind.version>
```

Comment:
Do we want to version jackson and jackson.databind separately?

Reply:
There is no 2.9.9.1 version for the other jackson dependencies used; there are only versions 2.9.9 and below.  I'll update jackson.version to 2.9.9.